src/app/visa/c.spec.ts

```diff
@@ -0,0 +1,630 @@
+            })
+            it('10 million', () => {
+                const checklist = simulationWithCriteriaC([
+                    annualSalaryOf(10_000_000), // 10 points
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(10)
+                expect(matches.map(m => m.id).sort()).toEqual(['10m_or_more'])
+            })
+            it('15 million', () => {
+                const checklist = simulationWithCriteriaC([
+                    annualSalaryOf(15_000_000), // 20 points
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(20)
+                expect(matches.map(m => m.id).sort()).toEqual(['15m_or_more'])
+            })
+            it('20 million', () => {
+                const checklist = simulationWithCriteriaC([
+                    annualSalaryOf(20_000_000), // 30 points
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(30)
+                expect(matches.map(m => m.id).sort()).toEqual(['20m_or_more'])
+            })
+            it('25 million', () => {
+                const checklist = simulationWithCriteriaC([
+                    annualSalaryOf(25_000_000), // 40 points
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(40)
+                expect(matches.map(m => m.id).sort()).toEqual(['25m_or_more'])
+            })
+            it('30 million', () => {
+                const checklist = simulationWithCriteriaC([
+                    annualSalaryOf(30_000_000), // 50 points
+                ])
+
+                const { matches, points } = calculatePoints(checklist)
+
+                expect(points).toBe(50)
+                expect(matches.map(m => m.id).sort()).toEqual(['30m_or_more'])
+            })
+        })
+
+        describe('position in company', () => {
+            it('representative director', () => {
+                const checklist = simulationWithCriteriaC([
+                    positionInCompany({ kind: 'representative_director' }), // 10 points
+                ])
+                const { matches, points } = calculatePoints(checklist)
+                expect(points).toBe(10)
+                expect(matches.map(m => m.id).sort()).toEqual(['representative_director'])
+
+            })
+            it('executive officer', () => {
+                const checklist = simulationWithCriteriaC([
+                    positionInCompany({ kind: 'executive_officer' }), // 5 points
+                ])
+                const { matches, points } = calculatePoints(checklist)
+                expect(points).toBe(5)
+                expect(matches.map(m => m.id).sort()).toEqual(['executive_officer'])
+
+            })
+            it('being both executive officer and representative director',
```

Comment:
```suggestion
            it('executive officer and representative director should not be mutually exclusive',
```

Describes desired outcome better.